R/tax_certainty.R

```diff
@@ -0,0 +1,33 @@
+#' Check if a taxon is certain or uncertain.
+#'
+#' Check whether a given taxon in a vector is certain or uncertain.
+#' Binomial names (genus and species) are considered "certain",
+#' whereas names containing only the genus,
+#' or genus and not determined species is considered "uncertain".
+#'
+#' @param taxon_names Character vector containing the species names to be
```

Comment:
As the function is currently implemented, I think the main advantages of this would be:
- Keeping data together and more organised
- Environment "tidier", i.e. less objects
- In line with a lot of the package

Any others?